addon/helpers/markdown-viewer.js

```diff
@@ -0,0 +1,33 @@
+import Ember from 'ember';
+import markdownit from 'markdown-it';
+
+const { htmlSafe } = Ember.String;
+
+export function markdownViewer(params) {
```

Comment:
Looking at this [comment](https://openscience.atlassian.net/browse/PRODUCT-242?focusedCommentId=45402&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-45402), it seems like this helper should limit which markdown capabilities are available. Headers, for example, are still allowed. Code, is still allowed.